src/components/CrisisLineItem.js

```diff
@@ -38,7 +38,7 @@ class CrisisLineItem extends React.Component {
                             <Link
                                 to={`/service/${object.slug}`}
                             >
-                                {object.site.name} {object.name}
+                                {object.site.name}
```

Comment:
In theory, yes. But in practice you get giant blocks of text with lots of repeated data. Really hard to read and the user would be more confused than informed.